tests/e2e/tidbcluster/across-kubernetes.go

```diff
@@ -152,35 +158,53 @@ var _ = ginkgo.Describe("[Across Kubernetes]", func() {
 			ginkgo.By("Deploy the basic cluster-3")
 			utiltc.MustCreateTCWithComponentsReady(genericCli, oa, tc3, 5*time.Minute, 10*time.Second)
 
-			ginkgo.By("Deploy status of all clusters")
+			ginkgo.By("Check status of all clusters")
 			err := CheckClusterDomainEffect(cli, []*v1alpha1.TidbCluster{tc1, tc2, tc3})
 			framework.ExpectNoError(err, "failed to check status")
-		})
 
-	})
+			// connectable test
+			_, err = utiltidb.TiDBIsConnectable(fw, tc1.Namespace, tc1.Name, "root", "")()
+			framework.ExpectNoError(err, "tc1 are not connectable")
 
-	ginkgo.Describe("[Advanced]", func() {
+			ginkgo.By("Scale in cluster-3, and delete the cluster-3")
 
-		// create three namespace
-		ginkgo.BeforeEach(func() {
-			ns1 := namespaces[0]
-			namespaces = append(namespaces, ns1+"-1", ns1+"-2")
+			_ = genericCli.Get(context.TODO(), types.NamespacedName{Namespace: tc3.Namespace, Name: tc3.Name}, tc3)
+			framework.ExpectNoError(controller.GuaranteedUpdate(genericCli, tc3, func() error {
+				tc3.Spec.PD.Replicas = 0
+				tc3.Spec.TiDB.Replicas = 0
+				tc3.Spec.TiKV.Replicas = 0
+				tc3.Spec.TiFlash.Replicas = 0
+				tc3.Spec.TiCDC.Replicas = 0
+				tc3.Spec.Pump.Replicas = 0
+				return nil
+			}), "failed to scale in cluster 3")
+			framework.ExpectNoError(genericCli.Delete(context.TODO(), tc3), "failed to delete cluster 3")
+			err = wait.PollImmediate(10*time.Second, 3*time.Minute, func() (bool, error) {
+				if err := CheckPeerMembersAndClusterStatus(genericCli, tc1, tc3); err != nil {
+					return false, nil
+				}
+				return true, nil
+			})
+			framework.ExpectNoError(err, "tc1 are not all healthy")
+			err = wait.PollImmediate(10*time.Second, 3*time.Minute, func() (bool, error) {
+				if err := CheckPeerMembersAndClusterStatus(genericCli, tc2, tc3); err != nil {
+					return false, nil
+				}
+				return true, nil
+			})
+			framework.ExpectNoError(err, "tc2 are not all healthy")
```

Comment:
Do we need to check data in DB to check cluster is working?

Reply:
Deleting a TC doesn't affect the data or TiDB functions. It may affect the status in TiDBCluster CR, stuck the operations in TiDB Operator.

Reply:
`Deleting a TC doesn't affect the data or TiDB functions` This is a requirement, we need to add a check to make sure the implementation meets this requirement.
`TiDBIsConnectable` should be good.

Reply:
We should also delete tc3 after this check according to the test plan?